core/server/data/migrations/versions/4.0/28-add-webhook-intergrations-foreign-key.js

```diff
@@ -0,0 +1,23 @@
+const logging = require('../../../../../shared/logging');
+const {createIrreversibleMigration} = require('../../utils');
+const {addForeign, dropForeign} = require('../../../schema/commands');
+
+module.exports = createIrreversibleMigration(async (knex) => {
+    logging.info('Adding the webhooks to integrations foreign key');
+
+    await dropForeign({
+        fromTable: 'webhooks',
+        fromColumn: 'integration_id',
+        toTable: 'integrations',
+        toColumn: 'id',
+        transaction: knex
+    });
+    await addForeign({
+        fromTable: 'webhooks',
+        fromColumn: 'integration_id',
+        toTable: 'integrations',
+        toColumn: 'id',
+        cascadeDelete: true,
```

Comment:
@tpatel I think we don't need cascading here :thinking:  the migration was meant to only introduce the foreign key as it did not exist before :wink:

Reply:
If so, we don't need the `dropForeign` either right?

Reply:
yes, exactly!

Reply:
having a little more thought about it... `ON DELETE CASCADE` (or as we call it in schema `cascadeDelete`) might be ok here because it would prevent us from having accidental orphaned webhooks when the integration is deleted!

Reply:
I'm still not sure `dropForeign` would serve any purpose here except of being a very defensive way of making sure there's no foreign key already. would like to hear why @tpatel added it in the first place :)

Reply:
Actually, we should clarify the naming of `cascadeDelete` or `cascade` - it was `cascade` in the previous PR

Reply:
What is the difference between `cascade` in the last PR vs `cascadeDelete` in this one? 😬

Reply:
yeah exactly I thought about an edge case where there would already be a foreign key, but it's not relevant here so I'm gonna remove the `dropForeign`.
The `ON DELETE CASCADE` seems relevant to avoid orphaned webhooks as you said, even more because there was an issue with orphaned webhooks first :)